app/messaging/email/create-submission-msg.js

```diff
@@ -248,6 +249,8 @@ function getScoreChance(rating) {
 
 function getEmailDetails(submission, desirabilityScore, rpaEmail, isAgentEmail = false) {
   const email = isAgentEmail ? submission.agentsDetails.emailAddress : submission.farmerDetails.emailAddress
+  const henJourney = submission.poultryType === getQuestionAnswer('poultryType', 'poultry-type-A1')
+  const PulletJourney = submission.poultryType === getQuestionAnswer('poultryType', 'poultry-type-A2')
```

Comment:
Not an actual issue per se, more for consistency: Could we make the variable name **pulletJourney** instead?